aiida/tools/importexport/dbimport/backends/common.py

```diff
@@ -108,3 +108,17 @@ def _sanitize_extras(fields: dict) -> dict:
     if fields.get('node_type', '').endswith('code.Code.'):
         fields['extras'] = {key: value for key, value in fields['extras'].items() if not key == 'hidden'}
     return fields
+
+
+def _strip_checkpoints(fields: dict) -> dict:
+    """Remove checkpoint from attributes of process nodes.
+
+    :param fields: the database fields for the entity
+    """
+    fields = copy.copy(fields)
```

Comment:
this should be moved inside the `if`, so we can save time/memory not having to copy the fields for every entity